application-services/custom/camera-management/docker-compose.yml

```diff
@@ -14,17 +14,39 @@
 # limitations under the License.
 #
 
-version: "3"
+version: '3.7'
```

Comment:
Probably not that important but the newest version is `3.8` right?

Reply:
Ok I looked into this more and seems like you kept this in sync with other edgex compose files.

Reply:
Not important.

Version 3.8
An upgrade of [version 3](https://docs.docker.com/compose/compose-file/compose-versioning/#version-3) that introduces new parameters. It is only available with Docker Engine version 19.03.0 and higher.

Introduces the following additional parameters:

[max_replicas_per_node](https://docs.docker.com/compose/compose-file/compose-file-v3/#max_replicas_per_node) in placement configurations
template_driver option for [config](https://docs.docker.com/compose/compose-file/compose-file-v3/#configs-configuration-reference) and [secret](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets-configuration-reference) configurations. This option is only supported when deploying swarm services using docker stack deploy.
driver and driver_opts option for [secret](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets-configuration-reference) configurations. This option is only supported when deploying swarm services using docker stack deploy.

Reply:
Good to know!